dev/com.ibm.ws.jaxws.cdi_fat/fat/src/com/ibm/ws/jaxws/cdi/FATSuite.java

```diff
@@ -10,17 +10,30 @@
  *******************************************************************************/
 package com.ibm.ws.jaxws.cdi;
 
+import org.junit.ClassRule;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 import org.junit.runners.Suite.SuiteClasses;
 
 import componenttest.custom.junit.runner.AlwaysPassesTest;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.RepeatTests;
 
 @RunWith(Suite.class)
 @SuiteClasses({
                 AlwaysPassesTest.class,
                 BasicTest.class,
 })
 public class FATSuite {
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
+                    .andWith(new FeatureReplacementAction().addFeature("jaxws-2.3")
+                                    .addFeature("cdi-2.0")
+                                    .removeFeature("cdi-1.2")
+                                    .removeFeature("jaxws-2.2")
+                                    .removeFeature("jsp-2.2")
+                                    .removeFeature("servlet-3.1")
+                                    .removeFeature("jaxwstest-2.2")
```

Comment:
I did not find any server.xml files that have jaxwstest-2.2 and jsp-2.2, so it seems unnecessary to have the removeFeature calls for them.

Reply:
removed

Reply:
One of the server.xml files in the project has servlet-3.1.  You removed that one as well.  Any reason why?